libfreerdp/codec/h264_mediacodec.c

```diff
@@ -86,13 +87,18 @@ typedef AMediaFormat* (*AMediaCodec_getInputFormat_t)(AMediaCodec*);
 const int COLOR_FormatYUV420Planar = 19;
 const int COLOR_FormatYUV420Flexible = 0x7f420888;
 
+const int MEDIACODEC_MINIMUM_WIDTH = 320;
```

Comment:
if not used outside this file, make these `static` (same for the existing ones above)